stairway/src/main/java/bio/terra/stairway/FlightState.java

```diff
@@ -104,7 +104,12 @@ public ProgressMeterReader getProgressMeters() {
     return progressMeters;
   }
 
+  /** Returns true if the flight is not in a terminal state. */
   public boolean isActive() {
```

Comment:
Using the same implementation as https://github.com/DataBiosphere/terra-workspace-manager/blob/0131b1e681bc8125428ec1aebf9030ee7e375b95/service/src/main/java/bio/terra/workspace/common/utils/FlightUtils.java#L274

Reply:
I see this is a public method.  When you make changes in WSM, if we still need to use this implementation could we instead call the Stairway method?

Reply:
I'm not following. I don't believe we will call this method at all.

Reply:
Also, would it be feasible to write a unit test for this, given that this is a bugfix?

Reply:
Ohhh, you are suggesting we call this method from WSM? Sure, I guess we could if we have updated WSM to use the new version of Stairway. I'm not sure how strongly I feel about it, but I could be convinced.

Reply:
Sure, I will write a unit test for this. I was thinking of this more as an example implementation, but now am coming around to it being possibly more widely used.

Reply:
> Ohhh, you are suggesting we call this method from WSM? Sure, I guess we could if we have updated WSM to use the new version of Stairway. I'm not sure how strongly I feel about it, but I could be convinced.

Yes, that's what I was getting at, rather than duplicating logic between services (it does seem like it makes sense to live in Stairway).

Reply:
Added test-- https://github.com/DataBiosphere/stairway/pull/137/commits/9312dd2211f8ae08a111ef40493964f35cbb233b